layouts/shortcodes/highlight.html

```diff
@@ -1,3 +1,4 @@
-<div class="m-text-highlight m-text-highlight--{{ (.Get 0) | markdownify }}" aria-label='{{ T (printf "highlight-%s" (.Get 0)) }}'>
-    {{ .Inner }}
+<div class="m-text-highlight m-text-highlight--{{ (.Get 0) | markdownify }}">
```

Comment:
```suggestion
<div class="m-text-highlight m-text-highlight--{{ .Get 0 }}">
```

I know that you didn't touch it in this PR, but the argument is just a simple string and not in markdown format. So there is no need to markdownify it.